src/OptiHPLCHandler/applications/empower_implementation/empower_tools.py

```diff
@@ -129,3 +129,23 @@ def determine_strong_eluent(gradient_table: List[dict]) -> Optional[str]:
             list_weak_eluents.append(composition)
 
     return strong_eluent, list_weak_eluents
+
+
+def determine_decreasing_weak_eluents(gradient_table: List[dict]) -> List[str]:
```

Comment:
Would it make sense to have one function that divides the elunets into strong, weak, and non-changing instead?